canonical-websites/www.ubuntu.com
# Resources page

## Done
Merge the new resources page

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/resources](http://0.0.0.0:8001/resources)
- Check everything works as expected
